platforms/core-runtime/internal-instrumentation-processor/src/test/groovy/org/gradle/internal/instrumentation/extensions/property/PropertyUpgradeCodeGenTest.groovy

```diff
@@ -363,6 +380,7 @@ class PropertyUpgradeCodeGenTest extends InstrumentationCodeGenTest {
             public abstract class Task {
                 @ReplacesEagerProperty(replacedAccessors = {
                     @ReplacedAccessor(value = AccessorType.GETTER, name = "getDestinationDir"),
+                    @ReplacedAccessor(value = AccessorType.GETTER, name = "addDestinationDir"),
```

Comment:
:thinking: I wonder if it makes sense to call `addDestinationDir` and  `destinationDir` a `GETTER` or `SETTER` when they are in fact methods only tangentially related to the replaced property. And do we need to rewrite them in the first place - don't we need to preserve these methods for source compatibility anyway?